api/export_pr_test.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/MakeNowJust/heredoc"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	diff "github.com/yudai/gojsondiff"
```

Comment:
It seems a bit heavy handed to require a new library just for a single test... I am wondering if there is a easier way to make the same assertions without requiring adding a whole new library to the project?